src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/AbstractEntityValidation.feature.disabled

```diff
@@ -1,18 +1,34 @@
 Feature: Client cannot POST an abstract entity
-  Ensure that clients cannot POST an abstract entity
+  Ensure that clients cannot interact with abstract entities
```

Comment:
Is it necessary that we have two lines above? Can we replace both line 1 and 2 with just one line? Maybe the following is a good description of the feature:

```
Feature: Reject client requests for abstract entities
```

Reply:
Done